tox.ini

```diff
@@ -12,15 +11,14 @@ basepython =
     python3.8: python3.8
     python3.9: python3.9
     python3.10: python3.10
+    python3.11: python3.11
 
 deps =
     django3.2: Django>=3.2,<4.0
     django4.0: Django>=4.0,<4.1
     django4.1: Django>=4.1,<4.2
-    wagtail2.15: wagtail>=2.15,<2.16  # LTS
-    wagtail2.16: wagtail>=2.16,<2.17
-    wagtail3.0: wagtail>=3.0,<3.1
-    wagtail4.0: wagtail>=4.0,<4.1
+    wagtail4.1: wagtail>=4.1,<4.2  # LTS
+    wagtail4.2: wagtail>=4.2rc1,<4.3
```

Comment:
```suggestion
    wagtail4.2: wagtail>=4.2,<5.0
```

Reply:
Hi @nickmoreton ,
I applied the suggestion. Thank you!